cli/azd/internal/cmd/add/add_configure_host.go

```diff
@@ -0,0 +1,256 @@
+package add
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"maps"
+	"os"
+	"path/filepath"
+	"slices"
+	"strings"
+
+	"github.com/azure/azure-dev/cli/azd/internal"
+	"github.com/azure/azure-dev/cli/azd/internal/appdetect"
+	"github.com/azure/azure-dev/cli/azd/internal/repository"
+	"github.com/azure/azure-dev/cli/azd/pkg/environment/azdcontext"
+	"github.com/azure/azure-dev/cli/azd/pkg/input"
+	"github.com/azure/azure-dev/cli/azd/pkg/output"
+	"github.com/azure/azure-dev/cli/azd/pkg/output/ux"
+	"github.com/azure/azure-dev/cli/azd/pkg/project"
+	"github.com/fatih/color"
+)
+
+func (a *AddAction) configureHost(
+	console input.Console,
+	ctx promptCtx) (*project.ServiceConfig, *project.ResourceConfig, error) {
```

Comment:
move the context to be the first arg

Reply:
Seems like you added it to all methods at the end.  Curious about why?

Reply:
Reordered parameters